src/views/Auth.vue

```diff
@@ -0,0 +1,54 @@
+<template lang="pug">
+  .nav-bar
+    .items-container
+      img.logo(src='../assets/wolox-logo.png')
+      button.logout-button(@click='logout')
+        | Logout
+
+</template>
+
+<script>
+import { deleteCurrentSession } from '@/services/auth'
+
+export default {
+  name: 'auth',
+  methods: {
+    logout () {
+      deleteCurrentSession()
+      this.$router.push('/login')
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../scss/commons/colors.scss";
+
+.nav-bar {
+  background-color: $white;
+  border-top: 5px solid $scooter;
+  box-shadow: 0 3px 5px $shadow;
+  width: 100vw;
```

Comment:
100%?

Reply:
Its parent is the div#app, which is being modified just by the scss `* { ... }`. Do you prefer I add a class to that div with a `width: 100%` and change this one to the same thing? Or should I leave it as it is now?